### DISCUSSION TITLE
Help with "rate to high"

### DISCUSSION THREAD

**Discussion by PStenbumling** [jomjol/AI-on-the-edge-device#1911] [General]
### Issue

it seems to get the recognition 
but there's a comma.
so 000002448, should be 000002.448 how do i configure this?
![image](https://user-images.githubusercontent.com/39261692/214074550-68c65a6f-0b7d-43eb-9638-8c3883baa899.png)

**Comment by caco3** [comment]
You have to set the ` DecimalShift` parameter.


BTW, your image is a bit blurry, I would suggest to try to improve the contrast!

**Reply by PStenbumling** [reply]
Got it!
Regarding the contrast, i need to remove "glue" in order to improve the contrast?

**Reply by caco3** [reply]
yes
